docs/web3.eth.rst

```diff
@@ -348,7 +348,7 @@ The following methods are available on the ``web3.eth`` namespace.
             'nonce': '0x3b05c6d5524209f1',
             'number': 2000000,
             'parentHash': '0x57ebf07eb9ed1137d41447020a25e51d30a0c272b5896571499c82c33ecb7288',
-            'receiptRoot': '0x84aea4a7aad5c5899bd5cfc7f309cc379009d30179316a2a7baa4a2ea4a438ac',
+            'receiptsRoot': '0x84aea4a7aad5c5899bd5cfc7f309cc379009d30179316a2a7baa4a2ea4a438ac',
```

Comment:
I believe this is technically a breaking change, but it is also a bugfix...  I'm inclined to just highlight this well in the release notes.  Alternative would be to duplicate this value into both versions of the key (plural and singular) and then remove the singular one in v6...

Reply:
I'd probably lean towards keeping both, but I don't have a strong preference either way. Maybe we could note that in the newsfragment?